src/app/dataValidator/helpers/validators/validateNode.test.js

```diff
@@ -209,26 +211,9 @@ describe('Validate block', () => {
 
   it('handleSchemaItems: should handle an array of blocks', () => {
     const dataWithBlocksArray = {
-      model: {
-        blocks: [
-          {
-            blockId: 'cafd1bff-a31b-03a8-a029-171eaad3e41a',
-            type: 'text',
-            model: {
-              blocks: [
-                {
-                  type: 'paragraph',
-                  blockId: 'ce0e3616-fe03-e3be-9da8-45461a0f726c',
-                  model: {
-                    text:
-                      'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
-                  },
-                },
-              ],
-            },
-          },
-        ],
-      },
+      model: modelTextBlock(
```

Comment:
Is there a reason we aren't using `singleTextBlock` here?

Reply:
@pjlee11 
This includes the object with the blocks array, which within it is `singleTextBlock`
```
{
  blocks: [singleTextBlock(text)]
}
```
This is the value needed for the model.